front-end/src/API/index.js

```diff
@@ -1,10 +1,27 @@
 import axios from 'axios';
 
 const baseURL = process.env.REACT_APP_BACKEND_URL;
+const token = getCookie('token');
+
+function getCookie(get_cookie) {
+  let name = get_cookie + '=';
+  let all_cookie = document.cookie.split(';');
+
+  for (let i = 0; i < all_cookie.length; i++) {
+    let your_cookie = all_cookie[i];
+
+    while (your_cookie.charAt(0) === ' ') your_cookie = your_cookie.slice(1);
+
+    if (your_cookie.indexOf(name) === 0)
+      return decodeURI(your_cookie.slice(name.length, your_cookie.length));
+  }
+}
```

Comment:
You might consider to use js-cookie next tim. It is easy to use and can get cookie with just 1 line of code
https://www.npmjs.com/package/js-cookie